### ISSUE TITLE
Readthedocs version selector pops up at unexpected side of screen

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/docs.nextstrain.org#76]
**Current Behavior**  
When I click on the readthedocs version selector, the popup appears not where I clicked (at the bottom left), but at the bottom right of the screen which is probably not how it's supposed to be.

**Expected behavior**  
The popup should open where one clicks. Maybe the position of the button was changed in css but not where the popup appears.

**Possible solution**  
Adjust css of popup?
Something like this
```
.rst-versions .rst-other-versions {
    font-size: 100%;
    padding:12px;
    color: gray;
    display: none;
}

.rst-versions.rst-badge {
    width: auto;
    left: 0px;
    right: auto;
    border: none;
    max-width: 500px;
}
```

**Your environment: if browsing Nextstrain online**  
- Brave

![image](https://user-images.githubusercontent.com/25161793/132353496-debb6eb6-f0ed-4d13-97c8-6c5507d11f04.png)

**Comment by tsibley** [comment]
I suspect this bug lies in the interaction between readthedocs.org and their sphinx-rtd-theme, possibly some version mismatch between the two and maybe triggered by the middleman of our own subclassed theme.

It is annoying to debug this precise sort of issue locally because it's not possible to (without immense effort) exactly replicate the RTD build process here. Best to use the PR builds to troubleshoot and debug it, instead.

**Comment by tsibley** [comment]
While in some RTD/theme code for other reasons today, I had an epiphany: the JS in step 2 above _tells_ the API what the theme is. Since the API doesn't rely on its own information in the decision it makes in step 3 and we control JS for our pages, we can lie in the RTD API call and get the right HTML injected. :-) Implemented in nextstrain/sphinx-theme#17.
